src/flamegraph/color/palettes.rs

```diff
@@ -59,6 +59,31 @@ pub(super) mod perl {
     }
 }
 
+pub(super) mod python {
+    use crate::flamegraph::color::BasicPalette;
+
+    fn split_any_path(path: &str) -> impl Iterator<Item = &str> {
+        path.split(|c| c == '/' || c == '\\')
+    }
+
+    pub(in super::super) fn resolve(name: &str) -> BasicPalette {
+      if split_any_path(name).any(|part| part == "site-packages") {
+            return BasicPalette::Mem;
+        } else if split_any_path(name).any(|part| {
+            part.strip_prefix("python")
+                .or_else(|| part.strip_prefix("Python"))
+                .map_or(false, |version| {
+                    version.chars().all(|c| c.is_digit(10) || c == '.')
+                })
+        }) || name.starts_with("<")
+        {
+            // stdlib
+            return BasicPalette::Gray
+        }
+        BasicPalette::Hot
```

Comment:
Same here as for `Mem` above — usually better to stick with single-color schemes.